supereeg/load.py

```diff
@@ -220,7 +220,12 @@ def _load_from_cache(fname, ftype, sample_inds=None, loc_inds=None, field=None):
         else:
             return Brain(**dd.io.load(fullpath))
     elif ftype is 'mo':
-        return Model(**dd.io.load(fullpath))
+        #return Model(**dd.io.load(fullpath))
+        num = _load_field(fullpath, field='numerator')
+        den = _load_field(fullpath, field='denominator')
+        locs = _load_field(fullpath, field='locs')
+        n_subs = _load_field(fullpath, field='n_subs')
+        return Model(data=np.divide(num, den), locs=locs, n_subs=n_subs)
```

Comment:
@lucywowen can you verify this is correct?  `num` and `den` are in log space, right?  shouldn't we be calling either `_to_exp_real` or doing the division in log space (i.e. something like `data = _to_exp_real(np.subtract(num, den))`)?

Reply:
Alternatively, can you just change line 228 to `return Model(numerator=num, denominator=den, locs=locs, n_subs=nsubs)`?

Reply:
This is the hack for the existing model objects. I think that _load_from_cache will only load the model objects that we added in the package.  If that's incorrect then yes, this would be a problem.

Reply:
Can we handle things differently based on the creation date of the brain objects?

Reply:
hmmm that sounds like a good idea - but just to be clear, its only model objects (not brain objects) that are affected

Reply:
changed to check daytime and if older than today, it does the hack